code/game/objects/items/weapons/storage/artistic_toolbox.dm

```diff
@@ -13,98 +13,127 @@
+						asize--
+					else
+						asize++
+					acount++
+				to_chat(user, "<span class='warning'>[up_and_down]</span>")
+				to_chat(user, "<i><b><font face = Tempus Sans ITC>His Grace accepts thee, spread His will! All who look close to the Enlightened may share His gifts.</font></b></i>")
+				original_owner = user
+				return
 	..()
 
 /obj/item/storage/toolbox/green/memetic/attackby(obj/item/I, mob/user)
 	if(activated)
-		if(istype(I, /obj/item/grab/))
+		if(istype(I, /obj/item/grab))
 			var/obj/item/grab/G = I
-			if(!G.affecting)
-				return
+			var/mob/living/victim = G.affecting
 			if(!user.HasDisease(new /datum/disease/memetic_madness(0)))
 				to_chat(user, "<span class='warning'>You can't seem to find the latch to open this.</span>")
 				return
-			if(!ishuman(G.affecting) || issmall(G.affecting))
-				to_chat(user, "<span class='warning'>His Grace will not accept such a meager offering!</span>")
+			if(!victim)
 				return
-			if(!mindless_override)
-				if(G.affecting.mind == null && G.affecting.ckey == null)
-					to_chat(user, "<span class='warning'>His Grace will not accept a soulless shell for an offering!</span>")
-					return
-			if(!G.affecting.stat && !G.affecting.restrained() && !G.affecting.weakened)
+			if(!victim.stat && !victim.restrained() && !victim.weakened)
 				to_chat(user, "<span class='warning'>They're moving too much to feed to His Grace!</span>")
 				return
-			user.visible_message("<span class='userdanger'>[user] is trying to feed [G.affecting] to [src]!</span>")
-			if(!do_mob(user, G.affecting, 30))
+			user.visible_message("<span class='userdanger'>[user] is trying to feed [victim] to [src]!</span>")
+			if(!do_mob(user, victim, 30))
 				return
-			G.affecting.forceMove(src)
-			user.visible_message("<span class='userdanger'>[user] has fed [G.affecting] to [src]!</span>")
 
-			consume(G.affecting)
+			user.visible_message("<span class='userdanger'>[user] has fed [victim] to [src]!</span>")
+
+			consume(victim)
 			qdel(G)
 
 			to_chat(user, "<i><b><font face = Tempus Sans ITC>You have done well...</font></b></i>")
 			force += 5
 			throwforce += 5
 			return
 
-	..()
+	return ..()
 
-/obj/item/storage/toolbox/green/memetic/proc/consume(mob/M)
-	if(!M)
+/obj/item/storage/toolbox/green/memetic/proc/consume(mob/living/L)
+	if(!L)
 		return
 	hunger = 0
 	hunger_message_level = 0
-	playsound(src.loc, 'sound/goonstation/misc/burp_alien.ogg', 50, 0)
-	M.emote("scream")
-	M.death()
-	M.ghostize()
-	if(M == original_owner)
-		qdel(M)
+	playsound(loc, 'sound/goonstation/misc/burp_alien.ogg', 50, 0)
+
+	if(L != original_owner)
+		var/list/equipped_items = L.get_equipped_items(TRUE)
+		if(L.l_hand)
+			equipped_items += L.l_hand
+		if(L.r_hand)
+			equipped_items += L.r_hand
+		if(equipped_items.len)
+			var/obj/item/storage/box/B = new(src)
+			B.name = "Box-'[L.real_name]'"
+			for(var/obj/item/SI in equipped_items)
+				L.unEquip(SI, TRUE)
+				SI.forceMove(B)
+			equipped_items.Cut()
```

Comment:
The equipment nonsense has to be done before the mob is `forceMove`'d to the toolbox, otherwise, it generates an unlogged runtime.